documentation/sphinx/source/api-c.rst

```diff
@@ -718,6 +724,10 @@ Applications must provide error handling and an appropriate retry loop around th
 
    Most applications will not call this function.
 
+.. function:: FDBFuture* fdb_transaction_get_approximate_size(FDBTransaction* tr)
+
+   Retrieves the approximate transaction size so far in the returned future, which is the summation of the estimated size of mutations, read conflict ranges, and write conflict ranges. The size can be obtained by calling :func:`fdb_future_get_int64()` with the returned :type:`FDBFuture*` object. This can be called multiple times before transaction is committed.
```

Comment:
This should use the same pattern as used in other functions returning futures (such as `fdb_transaction_get_versionstamp`.

Reply:
Done.